File: Week 6/sentimental-mario-less/mario.py
# Prompt the user for the half-pyramid's height
while True:
    height = int(input("Height: "))
    if 1 <= height <= 8:
        break

# Generate pyramid
for i in range(height):
    # Print spaces for the first half
    for j in range(height - i - 1):
        print(" ", end="")
    # Print hashes for the first half
    for j in range(i + 1):
        print("#", end="")
    # Print a new line
    print()

File: Week 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for height of pyramid
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Print out pyramid
    for (int i = 1; i <= height; i++)
    {
        // Print spaces
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }

        // Print hashes
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }

        // Print newline character
        printf("\n");
    }
}
